src/recipe.yaml

```diff
@@ -363,6 +363,22 @@
   extra_labels:
     openllm_alias: 70b,70b-instruct
     model_name: meta-llama/Meta-Llama-3.1-70B-Instruct
+'llama3.1-nemotron:70b-instruct-fp16':
+  project: vllm-chat
+  service_config:
+    name: llama3.1-nemotron
+    traffic:
+      timeout: 300
+    resources:
+      gpu: 1
+      gpu_type: nvidia-rtx-3060
```

Comment:
well 70B won't be able to run on 3060 right?

also we need to set tp for this model